### DISCUSSION TITLE
Welcome to cpeditor Discussions!

### DISCUSSION THREAD

**Discussion by coder3101** [cpeditor/cpeditor#706] [Development]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

### Have a question 😕  ?
* Check 😃  [Frequent asked questions](https://cpeditor.org/docs/faq)
* Maybe the 📜  [Documentation](https://cpeditor.org/docs) helps.
* If it is a general issue try  [Google](https://www.google.com) 🔍 .
* [ 🔎 in Q&A section](https://github.com/cpeditor/cpeditor/discussions?discussions_q=category%3AQ%26A) of the Github Discussion. Remember to 👍🏻, if it helped you.

If none of the above helps, open a new discussion under Q&A section and someone would help you. Remember to mark the thread resolved if it helped you. 👌🏻

**Comment by codewithvk** [comment]
is it have dark mode?

**Reply by coder3101** [reply]
Yes

**Reply by codewithvk** [reply]
Can you point in documentation?

**Reply by coder3101** [reply]
> Can you point in documentation?

https://cpeditor.org/docs

**Reply by ouuan** [reply]
1. You should [open a new discussion](https://github.com/cpeditor/cpeditor/discussions/new) instead of asking here.
2. The documentation you need: [UI Style](https://cpeditor.org/docs/preferences/appearance/#ui-style) and [Editor Theme](https://cpeditor.org/docs/preferences/appearance/#editor-theme).
